src/router/router.tsx

```diff
@@ -11,19 +11,28 @@ import MainPage from '@pages/main/page'
 import RefinerPage from '@pages/refiner/page'
 import WSPage from '@pages/ws/page'
 
-export const RouterBase = (): ReactElement => (
-  <Router>
-    <QueryParamProvider ReactRouterRoute={Route}>
-      <GlobalStyle />
-      <ToastContainer />
+export const RouterBase = (): ReactElement => {
+  let baseName = '/'
+  const featureId = window.location.pathname.match(/FOROME-\d*/gm)
```

Comment:
`m` and `g` modifiers are not needed here. `m` - for multiline, but `pathname` is single-line, and `g` for global match, but we need only first match